test/config.json

```diff
@@ -67,28 +67,6 @@
 			"RetryMax": 0,
 			"Tags": []
 		},
-		"merge_sharding": {
-			"File": "merge_sharding.py",
-			"Args": [],
-			"Command": [],
-			"Manual": false,
-			"Shard": 4,
```

Comment:
the process so far has been to move python tests to shard 5 (which means they don't run at all) instead of removing them. but I'll approve this since we are very close to deleting them.